test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -63,6 +65,35 @@
           (is (= "something-crazy/for-content-type"
                  (get-in resp [:request :content-type]))))))))
 
+(deftest code-id-injection-test
+  (testing "code_id is calculated and added to the catalog request."
+    (let [handler (fn ([req] {:request req}))
+          app     (build-ring-handler handler "1.2.3" (constantly "foo-is-my-codeid"))
+          resp    (app {:request-method :get
+                        :uri            "/v3/catalog/bar"})]
```

Comment:
Our style is to not align these things vertically.   https://github.com/puppetlabs/pl-clojure-style